tools/alpine/Makefile

```diff
@@ -4,27 +4,6 @@ ORG?=linuxkit
 IMAGE=alpine
 DEPS=packages
 
-# The logic for content trust is a bit convoluted because:
-# - The arm64 base image is currently not signed so we need to pull it
-#   with content trust disabled. This is controlled by
-#   DOCKER_CONTENT_PULL.
-# - 'docker build' with the FROM image supplied as environment
-#   variable *and* with DOCKER_CONTENT_TRUST=1 currently does not work
-#   (https://github.com/moby/moby/issues/34199). We therefor build
-#   with DOCKER_CONTENT_TRUST explicitly set to 0. However, we pull
-#   the base image just before with content trust enabled (if
-#   supported, see above).
-# - By default we always pull and push the linuxkit/alpine image with
-#   content trust, unless explicitly disabled with NOTRUST. Once the
-#   above issues are resolved, this will be the only mechanism to control
-#   content trust.
-ifdef NOTRUST
-DOCKER_CONTENT_PULL=0
-else
-DOCKER_CONTENT_PULL=1
-export DOCKER_CONTENT_TRUST=1
```

Comment:
This seems to completely disable DCT

Reply:
I can't see the necessary that we need to disable the DCT in any case given the issue we encountered before has been resolved (refer to the https://github.com/moby/moby/issues/34199), at that time, the Alpine base image is not multi-arch and not signed for AArch64, so we introduce the NO_TRUST as AArch64 specific workaround , but now it's not the case.

Reply:
I don't understand. `alpine:3.6` *is* signed. So is the `linuxkit/alpine`. So everything should have content trust enabled, and if my reading of this patch is correct it disables it

Reply:
Ah, if the default `docker build` behavior is DCT disabled, then you're right. is it?
But anyway, set DOCKER_CONTENT_TRUST=1 before the following `docker build` in this Dockerfile will pretty ease your concern here.  I give the previous comments because you quoted code is not in `docker build` followed. Will force update soon.

Reply:
Actually IMO we don't need an explicit `DOCKER_CONTENT_TRUST=1' prefixed before `docker build --no-cache --iidfile iid .`, just as you pointed: both `alpine:3.6` and `linuxkit/alpine` are or will be signed.

Reply:
AIUI without `DOCKER_CONTENT_TRUST=1` nothing will check the signatures, so it is needed. Please do the same as `kernel/Makefile` which is:
```
ifeq ($(DOCKER_CONTENT_TRUST),)
ifndef NOTRUST
export DOCKER_CONTENT_TRUST=1
endif
endif
```
And remove the other settings of `DOCKER_CONTENT_TRUST` in this `Makefile`.

`NOTRUST` is useful for people doing local or development builds, perhaps in other orgs or with other base images.

Reply:
Just googled the DCT purpose, `Once you set this environment variable, every Docker operation is secure. This means that any future Docker commands are executed based on signatures.`, so my understanding is, we need to use 
```
ifeq ($(DOCKER_CONTENT_TRUST),)
ifndef NOTRUST
export DOCKER_CONTENT_TRUST=1
endif
endif
```
the same block as kernel  into this PR's and remove other DCT variable prefixed.
Right? @ijc

Reply:
> Right? @ijc

Since you repeated exactly what I said I of course agree.